static/js/NavSidebar.jsx

```diff
@@ -8,14 +8,18 @@ import { Promotions } from './Promotions'
 import {SignUpModalKind} from "./sefaria/signupModalContent";
 
 const NavSidebar = ({modules}) => {
-  return <div className="navSidebar sans-serif">
+  return (
+    <div className="navSidebar sans-serif">
     {modules.map((m, i) =>
       <Modules
         type={m.type}
         props={m.props || {}}
         key={i} />
     )}
+    <SidebarFooter />
```

Comment:
Do we want to allow for the possibility that someone chooses to render the NavSidebar in a non default way without a footer?

Reply:
Hmm possibly, what's the best way to do that? Control via props?